core/domain/question_domain.py

```diff
@@ -82,9 +82,7 @@ def __init__(self, change_dict):
 
         if self.cmd == CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION:
             self.question_dict = change_dict['question_dict']
-            self.skill_id = None
-            if 'skill_id' in change_dict:
-                self.skill_id = change_dict['skill_id']
+            self.skill_id = change_dict['skill_id']
```

Comment:
Not a bad idea to add a comment above this line saying "Note that change_dict['skill_id'] may be None if this change is being done in the context of a suggestion."

Reply:
Done